Extensive reorg/expansion of workspaces and runs sections (#1078)
hashicorp/terraform-website: content/source/docs/cloud/run/index.html.md

### Configuration Versions

Each workspace is associated with a particular Terraform configuration, but that configuration is expected to change over time. Thus, Terraform Cloud manages configurations as a series of _configuration versions._

Most commonly, a workspace is linked to a VCS repository, and its configuration versions are tied to revisions in the specified VCS branch. In workspaces that aren't linked to a repository, new configuration versions can be uploaded via Terraform CLI or via the API.

### Ordering and Timing

Each workspace in Terraform Cloud maintains its own queue of runs, and processes those runs in order.

Whenever a new run is initiated, it's added to the end of the queue. If there's already a run in progress, the new run won't start until the current one has completely finished — Terraform Cloud won't even plan the run yet, because the current run might change what a future run would do. Runs that are waiting for other runs to finish are in a _pending_ state, and a workspace might have any number of pending runs.

> When you initiate a run, Terraform Cloud locks the run to a particular configuration version and set of variable values. If you change variables or commit new code before the run finishes, it will only affect future runs, not runs that are already pending, planning, or awaiting apply.

Reviewer: > to a particular configuration version and set of variable

add `a` after `and`

>to a particular configuration version and **a** set of variable

Author: Not grammatically necessary here. The distribution over the groups ends up resolving like so:

> to a particular ((configuration version) and (set of variable values))

> to (a particular configuration version) and (a particular set of variable values)

## Run States

> Each run passes through several stages of action (pending, plan, policy check, apply, and completion), and Terraform Cloud shows the progress through those stages as run states. In some states, the run might require confirmation before continuing or ending; see [Managing Runs: Interacting with Runs](./manage.html#interacting-with-runs) for more information.

Reviewer: > Each run passes through several stages of action (pending, plan, policy check, apply, and completion), and Terraform Cloud shows the progress through those stages as run states.

Terraform Cloud shows the progress of each run as it passes through each run state (pending, plan, policy check, apply, and completion).

Author: Ah, good call. 

Hey, have you seen the "suggestions" interface yet? It's a button with a page that has a plus/minus icon on it, hidden next to the markup helper buttons:

```suggestion
Terraform Cloud shows the progress of each run as it passes through each run state (pending, plan, policy check, apply, and completion). In some states, the run might require confirmation before continuing or ending; see [Managing Runs: Interacting with Runs](./manage.html#interacting-with-runs) for more information.
```

Reviewer: Wasn't aware that was there, thanks! That'll come in handy :)
